### ISSUE TITLE
Set screws in the rotor base are not mentioned

### ISSUE THREAD

**Issue by ablot** [jonnew/twister3#4]
Small documentation point:

The three threads in the rotor based that are presumably made for set screws that hold the alignment plate in place are not mentionned in the manuscript.

It would be nice to mention that they should be put before trying to load the bobin or before twisting the tetrode (and they should be untightened to align)

**Comment by jonnew** [comment]
This is correct. Actually I had forgotten about those because our fits were tight enough that we never had to use them. But, as 3D print tolerances are hard to control we put them in there just in this case :). When we create the docs site, we will be sure to mention this. Thanks for the report.
